content/docs/setup/kubernetes/prepare/platform-setup/gardener/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Gardener - Managed Kubernetes clusters
```

Comment:
How about just calling it "Kubernetes Gardener"?

Reply:
Done